wordCounter.js

```diff
@@ -0,0 +1,41 @@
+console.log(process.argv);
+if(process.argv.length === 2) {
+    console.log("please enter argv"); 
```

Comment:
It is good to write more detailed information in console. Something like : "Wrong number of arguments, usage : "